src/libcore/result.rs

```diff
@@ -685,7 +685,7 @@ impl<T, E: fmt::Debug> Result<T, E> {
         match self {
             Ok(t) => t,
             Err(e) =>
-                panic!("called `Result::unwrap()` on an `Err` value: {:?}", e)
+                unwrap_failed("called `Result::unwrap()` on an `Err` value: {:?}", e),
```

Comment:
It's supposed to be more consistent with a trailing comma for every case. Not sure what rustfmt usually does.